addons/payment/models/payment_token.py

```diff
@@ -125,6 +125,19 @@ def _build_display_name(self, *args, max_length=34, should_pad=True, **kwargs):
             display_name = self.payment_details[-max_length:] if max_length > 0 else ''
         return display_name
 
+    def _get_available_tokens(self, *args, **kwargs):
+        """ Return the available tokens linked to a context defined in args/kwargs.
+
+        For a module to retrieve the linked tokens, it must override this method and add information
+        in the args/kwargs to define the context of the request.
+
+        :param tuple args: Locally unused arguments.
```

Comment:
I wouldn't bother supporting `args` in the signature. Most of the time, one will pass a kwarg to the method and extract it in an override; and in the case where a positional argument is currently being passed, it could be converted to a kwarg for more simplicity.

Discord this comment if it doesn't apply.